site-src/implementations.md

```diff
@@ -0,0 +1,95 @@
+
+- :heavy_check_mark: available
+- :heavy_plus_sign: in progress
+- :heavy_minus_sign: not planned
+
+[1]:#contour
+[2]:#google-cloud-platform
+[3]:#istio
+[4]:#kong
+[5]:#knative
+[6]:#traefik
+
+[v0.1]:https://github.com/kubernetes-sigs/gateway-api/releases/tag/v0.1.0
+[v0.2]:https://github.com/kubernetes-sigs/gateway-api/releases/tag/v0.2.0
+
+## Project References
+
+In this section you will find specific links to blog posts, documentation and other Gateway API references for specific implementations.
+
+### Contour
+
+[Contour][contour] is an open source ingress controller for Kubernetes.
+
+Contour currently implements the latest `v0.2.x` Gateway API Specification.
+
+See the [Contour Gateway API Guide][contour-1] for information on how to deploy and use Contours's Gateway implementation.
+
+[contour]:https://projectcontour.io
+[latest]:https://doc.traefik.io/traefik/routing/providers/kubernetes-gateway/
+[contour-1]:https://projectcontour.io/guides/gateway-api/
+
+### Google Cloud Platform
+
+The [Google Cloud Platform (GCP)][gcp] is a cloud computing platform and infrastructure provider.
+
+GCP is actively working towards an implementation of the Gateway API `v0.2.x` specification, and status updates and documentation will be provided here as the work progresses.
+
+[gcp]:https://cloud.google.com
+
+### Istio
+
+[Istio][istio] is an open source [service mesh][mesh] for Kubernetes clusters.
+
+Istio currently supports the previous `v0.1.x` Gateway API specification and is working towards supporting the latest `v0.2.x` release.
+
+Status updates and documentation for `v0.2.x` will be provided here as the work progresses.
+
+See the [Istio Gateway API Documentation][istio-1] for information on how to deploy and use Istio's Gateway implementation.
+
+[istio]:https://istio.io
+[mesh]:https://istio.io/latest/docs/concepts/what-is-istio/#what-is-a-service-mesh
+[istio-1]:https://istio.io/latest/docs/tasks/traffic-management/ingress/service-apis/
+
+### Kong
+
+[Kong][kong] is an open source API Gateway built for hybrid and multi-cloud environments.
+
+Kong is actively working towards an implementation of the Gateway API `v0.2.x` specification for it's [Kubernetes Ingress Controller][kic] and status updates and documentation will be provided here as the work progresses.
+
+You can subscribe to [kong/kubernetes-ingress-controller/issues/692][kong-1] to track the implementation progress and contribute.
+
+[kong]:https://konghq.com
+[kic]:https://github.com/kong/kubernetes-ingress-controller
+[kong-1]:https://github.com/kong/kubernetes-ingress-controller/issues/692
+
+### Knative
+
+[Knative][knative] is an open source Kubernetes-based platform to deploy and manage modern serverless workloads.
+
+Knative is actively working towards an implementation of the Gateway API, status updates and documentation will be provided here as the work progresses.
```

Comment:
I think its a bit misleading? Knative is a user of the gateway API, not an implementation, afaik

Reply:
Roger that, I'll find someone from Knative and work on getting this written more appropriately for their liking :+1:

Reply:
Yeah, as @howardjohn also mentioned, Knative is an user of the gateway API but not an implementation. Knative is going to support the implemented Ingress like Istio, Kong, Contour etc.

Reply:
I've removed Knative for now as per above comments: 7781c9acbc944c0c05270bc322bd2ba24509a17c

We can come back around to documenting it further in a later iteration :+1: